reports/tasks.py

```diff
@@ -127,3 +106,83 @@ def send_trial_instances_report(recipients=settings.TRIAL_INSTANCES_REPORT_RECIP
     email.attach_file(csv_filename)
 
     return email.send()
+
+
+# Tasks #######################################################################
+
+# Run on the 1st of every month
+@db_periodic_task(
+    crontab(
+        minute=TRIAL_INSTANCES_REPORT_SCHEDULE_MINUTE,
+        hour=TRIAL_INSTANCES_REPORT_SCHEDULE_HOUR,
+        day=TRIAL_INSTANCES_REPORT_SCHEDULE_DAY,
+        month=TRIAL_INSTANCES_REPORT_SCHEDULE_MONTH,
+        day_of_week=TRIAL_INSTANCES_REPORT_SCHEDULE_DAY_OF_WEEK
+    )
+)
+def send_trial_instances_report(recipients=settings.TRIAL_INSTANCES_REPORT_RECIPIENTS):
```

Comment:
@janimo, can you add/update the tests in `reports/tests.py` (`ReportTaskTestCase` class) for the changes made in this PR?

Reply:
I have now added a few tests in that file.